apps/frontend/src/modules/workspace/components/settings/Workspace/components/MemberList.tsx

```diff
@@ -0,0 +1,212 @@
+import { OrganizationMemberCountableEdge, RoleLevel, User } from "@/generated/graphql";
+import { useAuth } from "@/modules/auth/hooks/useAuth";
+import { convertToAuthOrganization } from "@/modules/auth/states/user";
+import { useWorkspace } from "@/modules/workspace/hooks/useWorkspace";
+import { useWorkspaceInvite } from "@/modules/workspace/hooks/useWorkspaceInvite";
+import { ROUTES } from "@/routes";
+import { GlobalSpinner } from "@/ui";
+import { DeepPartial } from "@apollo/client/utilities";
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import { MoreHorizontal } from "lucide-react";
+import { useEffect, useMemo, useState } from "react";
+import toast from "react-hot-toast";
+import { useNavigate } from "react-router-dom";
+
+type Props = {
+    searchValue: string;
+};
+
+export const MemberList = ({ searchValue }: Props) => {
+    const navigate = useNavigate();
+    const { workspace } = useWorkspace();
+    const { user, switchWorkspace, organizations, updateUser } = useAuth();
+    const { removeOrganizationMember, leaveOrganization, updateMemberRole, loading } = useWorkspaceInvite();
+
+    const [filteredMembers, setFilteredMembers] = useState<OrganizationMemberCountableEdge[]>([]);
+
+    useEffect(() => {
+        if (searchValue) {
+            const filtered = workspace?.members?.edges?.filter((member) =>
+                `${member?.node?.firstName} ${member?.node?.lastName}`
+                    .toLowerCase()
+                    .includes(searchValue.toLowerCase()),
+            );
+            setFilteredMembers(filtered as OrganizationMemberCountableEdge[]);
+        } else {
+            setFilteredMembers(workspace?.members?.edges as OrganizationMemberCountableEdge[]);
+        }
+    }, [searchValue, workspace?.members?.edges]);
+
+    const handleLeaveOrganization = async (organizationId: string) => {
+        const response = await leaveOrganization(organizationId);
+        console.log(organizationId, response);
```

Comment:
@afolasope This too. Also, check all occurrences of `console.log`, then remove them.

Reply:
I have removed them